src/app/components/chat/room-channel.ts

```diff
@@ -7,7 +7,7 @@ import {
 	onUnmounted,
 	ref,
 	shallowReadonly,
-	watch,
```

Comment:
Looks like you don't have one of the extensions installed that adds a tailing comma to these imports.